Doc 252 remove nginx agent docs (#84)
nginxinc/nginx-amplify-doc: content/metrics-metadata/nginx-metrics.md

## Additional NGINX Metrics

> NGINX Amplify Agent can collect many additional useful metrics described below. To enable these metrics, please make the following configuration changes. More predefined graphs will be added to the **Graphs** page if NGINX Amplify Agent finds additional metrics. With the required log format configuration, you'll can build more specific custom graphs.

Reviewer: ```suggestion
NGINX Amplify Agent can collect many useful metrics, as described below. To enable these metrics, make the following configuration changes. More predefined graphs will be added to the **Graphs** page if NGINX Amplify Agent finds additional metrics. With the required log format configuration, you can build more specific custom graphs.
```
